doc/chsmack.8

```diff
@@ -26,7 +26,7 @@ chsmack \- Change or list the Smack properties of filesystem objects
 
 .B chsmack [-L|--dereference] [--] files...
 
-.B chsmack [-L|--dereference] props... [--] files...
+.B chsmack [-L|--dereference] [-D|--drop] props... [--] files...
```

Comment:
For sure. But why to not keep that facility? It is good to write "chsmack -D file" it drops all.